packages/cli/src/commands/generate/scaffold/templates/assets/scaffold.css.template

```diff
@@ -1,232 +1,82 @@
-/*
-  This file was created automatically as the result of the scaffold generator.
-  If you decide to use TailwindCSS in your project then you can safely delete
-  this file, but first remove the <div className="rw-scaffold"> from the Layout
-  that your scaffold is wrapped in. Check the comment blocks below for a
-  couple of config options you'll need to add to Tailwind to preserve the
-  scaffold appearance.
-
-  If you don't use Tailwind then you should keep it here until you have
-  completely converted your scaffolded pages or removed them, or if you don't
-  mind your scaffolded pages being monumentally ugly.
-*/
-
 /*
   normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css
 */
-
-.rw-scaffold main{display:block}
-.rw-scaffold h1{font-size:2em;margin:.67em 0}
-.rw-scaffold a{background-color:transparent}
-.rw-scaffold input{font-family:inherit;font-size:100%;line-height:1.15;margin:0}
-.rw-scaffold input{overflow:visible}
-.rw-scaffold *,.rw-scaffold ::after,.rw-scaffold ::before{box-sizing:inherit}
-.rw-scaffold h1,.rw-scaffold h2{margin:0}
-.rw-scaffold ul{list-style:none;margin:0;padding:0}
-.rw-scaffold *,.rw-scaffold ::after,.rw-scaffold ::before{border-width:0;border-style:solid;border-color:#e2e8f0}
-.rw-scaffold input:-ms-input-placeholder{color:#a0aec0}
-.rw-scaffold input::-ms-input-placeholder{color:#a0aec0}
-.rw-scaffold input::placeholder{color:#a0aec0}
-.rw-scaffold table{border-collapse:collapse}
-.rw-scaffold h1,.rw-scaffold .rw-scaffold h2{font-size:inherit;font-weight:inherit}
-.rw-scaffold a{color:inherit;text-decoration:inherit}
-.rw-scaffold input{padding:0;line-height:inherit;color:inherit}
+*,::after,::before{box-sizing:inherit;border-width:0;border-style:solid;border-color:#e2e8f0}
```

Comment:
Hmmm now I'm torn about what to do here. These normalize rules will now apply site-wide to any and all of the tags listed. :( So as soon as you scaffold your first model, your entire app's appearance could change. 

This makes me think that maybe we *should* keep the `rw-scaffold` wrapper, but only to make these normalize rules specific enough that they don't bother the rest of the site. All the other styles can still target `rw-link`, `rw-heading` etc. without them being subclassed under `rw-scaffold`. 

What do you think?

Reply:
@cannikin Agreed. I fix.